api/caaCaseTypes/caaCaseTypes.util.ts

```diff
@@ -28,11 +40,17 @@ export function getRequestBody(organisationID: string, caaCasesPageType: string)
                 must_not: [
                   { range: { [organisationAssignedUsersKey]: { gt: 0 } } }
                 ]
+              })
+            }
+          },
+          {
+            bool: {
+              ...(caaCasesFilterValue && !Array.isArray(caaCasesFilterValue) && {
+                must: caseReferenceFilter
```

Comment:
@johnbenjamin-hmcts - just checking that the use of `must` on Line 49 and `should` on Line 52 is intentional

Reply:
@Daniel-Lam - we have implemented the same logic when searching for cases. The filter keyword 'must' is when searching for a single case reference and 'should' is when searching for multiple case references.